### ISSUE TITLE
Cannot pair devices

### ISSUE THREAD

**Issue by ijontek** [dead/moonlight-openelec-rpi2#14]
Hey,

Firstly, great work! I love the speed of openelec and I was thinking that it would be great to add the option of streaming from PC. Anyways, let me put some background:
1. I had an issue with installing the addon, however re-zipping it, as suggested in this [thread ](https://github.com/dead/moonlight-openelec-rpi2/issues/10), did the trick and eventually moonlight was installed,
2. After trying to run Moonlight via Kodi dashboard, I experienced something similiar to what was described in this [thread](https://github.com/dead/moonlight-openelec-rpi2/issues/2), namely the Kodi itself restarted (not the RPi2), when I typed the pin to PC and attempted to connect devices.
3. So, I ssh'd to openelec and tried to manually pair moonlight with `moonlight pair XXX.XXX.XXX.XXX`

And this is where I'm stuck. The result of the command was `Please enter the following PIN on the target PC: 5742
*** Error in 'moonlight': free(): invalid pointer: 0x002fbda0 ***
Aborted` 

What should I do?

Cheers

**Comment by dead** [comment]
What OpenElec version are you using? 6 or 7? When running the moonlight via Kodi, the game list appears? And please send the kodi.log when you try to open it. Thanks!

**Comment by ijontek** [comment]
Thanks for the quick response, I use openelec 5.0.2.  I just read that it should be under version 6 and was about to update, but I found a workaround. Here is what I did:
1. De-installed the addon, which had the issue described.
2. Download [this folder](https://github.com/dead/moonlight-openelec-rpi2/blob/master/script.moonlight.zip) exactly 
3. Installed via dashboard from the downloaded zip placed in `/storage/downloads`
4. Added `libcrypto.so.32` & `libssl.so.32` to `~/.kodi/addons/script.moonlight/lib`
5. Success!

Moonlight was able to pair in first attempt. Works great!
Thanks for the support!
